ui/apps/platform/src/Containers/NetworkGraph/TopologyComponent.tsx

```diff
@@ -138,7 +138,7 @@ const TopologyComponent = ({
     }, [controller]);
 
     const panNodeIntoView = useCallback(
-        (node: any) => {
+        (node) => {
```

Comment:
Can this work instead? It looks like this will avoid an addition of an `implicit-any`
```suggestion
        (node: CustomNodeModel) => {
```

Reply:
Yes, super! Thank you.